backend/controller/sql/schema/20240815164808_async_calls_cron_job_key.sql

```diff
@@ -0,0 +1,24 @@
+-- migrate:up
+
+ALTER TABLE async_calls
+    ADD COLUMN cron_job_key cron_job_key;
```

Comment:
This is not great database design, we need to change this. This is akin to having a parent class and subclass, and the parent class contains a reference to the subclass. It needs to be inverted so the cron table references the async_calls table.

Reply:
Sounds good, that was one of the options I proposed in the design doc, and I only just noticed your thumbs-up on that 🙃